14/umbraco-cms/customizing/section-trees/sections/section-view.md

```diff
@@ -12,17 +12,100 @@ This page is a work in progress and may undergo further revisions, updates, or a
 
 <figure><img src="../../../.gitbook/assets/section-views.svg" alt=""><figcaption><p>Section View</p></figcaption></figure>
 
-**Manifest**
+## Creating a custom Section View
+
+### Manifest
+
+**Using Json**
+
+We can create the manifest using json in the `umbraco-package.json`.
 
 ```json
 {
 	"type": "sectionView",
 	"alias": "My.SectionView",
 	"name": "My Section View",
+	"element": "./my-section.element.js",
 	"meta": {
-		"sections": ["My.Section"],
 		"label": "My View",
-		"pathname": "/my-view"
-	}
+		"icon": "icon-add",
+		"pathname": "my-view"
+	},
+	"conditions": [
+		{
+			"alias": "Umb.Condition.SectionAlias",
+			"match": "My.Section",
+		}
+	]
+}
+```
+
+**Using TypeScript**
```

Comment:
@Yinzy00 this section where the manifest file is made with TS, is it implicit that you are using the bundle type?
https://docs.umbraco.com/umbraco-cms/customizing/extending-overview/extension-types/bundle

Reply:
I'm not using bundle i'm using entrypoint.

Reply:
You want me to clarify more?

Reply:
Yes please 😄

Reply:
I’ve added more context about the project setup for the example. However, I’m not sure if this belongs on this page since it seems to relate more to entrypoint extensions. What do you think?

Reply:
I agree, what about doing a ref to the entry point extension with something like if you want to know more about the entry point method read it here

Reply:
Done ;)